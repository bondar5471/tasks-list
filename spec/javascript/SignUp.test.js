import React from 'react';
import  renderer from 'react-test-renderer'
import { BrowserRouter as Router } from 'react-router-dom'

import SigmUp from '../../src/components/sign-up/sign-up';

it('renders without crashing', () => {
    const Sign = renderer.create(
        <Router><SigmUp /></Router>).toJSON()
    expect(Sign).toMatchSnapshot();

});
