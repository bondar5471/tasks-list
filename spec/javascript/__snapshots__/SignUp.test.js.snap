// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="Login"
>
  <form
    onSubmit={[Function]}
  >
    <div
      className="form-group"
    >
      <label>
        Email
      </label>
      <input
        autoFocus={true}
        className="form-control"
        id="authEmail"
        onChange={[Function]}
        placeholder="Email addres"
        type="email"
      />
    </div>
    <div
      className="form-group"
    >
      <label>
        Password
      </label>
      <input
        className="form-control"
        id="authPassword"
        onChange={[Function]}
        placeholder="Password"
        type="password"
      />
    </div>
    <div
      className="form-group"
    >
      <label>
        Password confirmation
      </label>
      <input
        className="form-control"
        id="authPasswordConfirmation"
        onChange={[Function]}
        placeholder="Password confirmation"
        type="password"
      />
    </div>
    <button
      className="btn btn-primary btn-block"
      disabled={true}
      type="submit"
    >
      Sign Up
    </button>
    <a
      className="link-login"
      href="/login"
      onClick={[Function]}
    >
      Want to login
    </a>
  </form>
</div>
`;
